website/versioned_docs/version-v3.11.x/exempt-namespaces.md

```diff
@@ -69,4 +69,6 @@ The difference is at what point in the admission process an exemption occurs.
 
 If you use `--exempt-namespace` flag and `admission.gatekeeper.sh/ignore` label, Gatekeeper's webhook will not be called by the API server for any resource in that namespace. That means that Gatekeeper being down should have no effect on requests for that namespace.
 
+> In order to add the `admission.gatekeeper.sh/ignore` label to a namespace, that namespace must be listed under the gatekeeper `controllerManager.exemptNamespaces` parameter.
```

Comment:
please add these changes to the rest of the versioned docs too since it should apply for all of them